docs/CoC.md

```diff
@@ -0,0 +1,211 @@
+principles above, in the spirit in which they are included in this Code of
+Conduct.
+
+## Expected behaviour
+
+The following behaviours are expected and requested of all community members:
+
+* Participate in an authentic and active way. In doing so, you contribute to the
+  health and longevity of this community.
+* Exercise consideration and respect in your speech and actions.
+* Attempt collaboration before conflict.
+* Refrain from demeaning, discriminatory and harassing behaviour and speech.
+* Be mindful of your surroundings and your fellow participants. Alert community
+  leaders if you notice a dangerous situation, someone in distress, or
+  violations of this Safety and Well-Being Policy, even if they seem
+  inconsequential.
+* Remember that community venues, both physical and digital, may be shared with
+  members of the public. Please be respectful to all patrons of such locations.
+
+## Unacceptable behaviour
+
+The following behaviours are considered harassment and are unacceptable within
+our community:
+
+* Violence, threats of violence or violent language directed against another
+  person. Note that misgendering, especially intentional misgendering, is a
+  form of violent language and will not be tolerated.
+* Sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory
+  jokes and language.
+* Posting or displaying sexually explicit or violent material.
+* Posting or threatening to post other people's personally identifying
+  information ("doxing").
+* Personal insults, particularly those related to gender, sexual or romantic
+  orientation, ethnicity, religion or disability.
+* Inappropriate photography or recording. Note that photography or recording
+  without prior consent is _always_ inappropriate.
+* Inappropriate physical contact. You must have someone's consent before
+  touching them.
+* Unwelcome sexual attention. This includes, but is not limited to: sexualized
+  comments or jokes, inappropriate touching, groping and unwelcome sexual
+  advances. If you do not have someone's prior consent, then any kind of sexual
+  attention is unwelcome.
+* Deliberate intimidation, stalking or following (online or in person).
+* Advocating for, or encouraging, or assisting in any of the above behaviour.
+* Sustained disruption of community spaces or events.
+
+## Consequences of unacceptable behaviour
+
+Unacceptable behaviour from any community member, including anyone with
+decision-making authority, will not be tolerated. Anyone asked to stop
+unacceptable behaviour is expected to comply immediately.
+
+If a community member engages in unacceptable behaviour, the community
+organizers may take any action they deem appropriate. These include, but are not
+limited to:
+
+* A verbal warning (either in private or in public).
+* A temporary ban of any length with right of appeal.
+* A temporary ban of any length without right of appeal, by unanimous consent of
+  the community organizers.
```

Comment:
This feels a bit... problematic to me, as Pext is pretty much only my project, and I don't want people to go and bother others because I won't listen to them. I'd rather have "with consent of the lead organizer" there. I know it is a bit of a "power grab" on my side, but well, it's my project in the end, and I want to be able to make the final decisions on abusive people.

Reply:
'Who watches the watchmen?' is a _very_ important question to ask with anything like this. You and I have both witnessed lead meltdowns which caused severe damage to the credibility and community of several projects. I don't want to assume that this will happen to you, but I prefer to have strong institutions in case something like this does happen. I will make this change if you request it, but I'm not sure I'd be good to serve on a community organization team in such a case.

Reply:
Well, I do understand that, but I do really want to stay in control of my own project, especially because practically 100% of the code, documentation and everything (except translations and themes) is written and maintained by me. I don't want to end up risking being alienated from my own project by ending in arguments on someone of the team unbanning someone I really want gone. If it really gets too bad, a hard fork as always possible. If enough of the community agrees they can always push me out of it. If you don't want to be part of it in this case, that's up to you.

Reply:
I have to agree on her argument to make final decisions as this is her project. She does reserve that right. If it was my project, I'd also be completely against sharing that power.

Reply:
Just to clarify @TheLastProject, do you want decision control over _non-appealable_ enforcement actions, or _all_ enforcement actions? I'm not 100% clear on this.

Reply:
Honestly, I want decisive control over every action. That doesn't mean I need to be consulted before every action. I just want some logs of who did what when for what reason (like, a Riot channel or GitHub team or whatever), so I know what's happening and can talk things through with a mod or reverse their action if I feel it's appropriate. Regardless of me reversing something, I feel we should have some central place where all moderation actions are logged anyway.

I know you're looking to create a democracy, but as long as still practically 100% of the entire project's code is mine, I am not ready to share Pext. The most share anyone else has in this project is @VenomousDingo by having created all themes and managing the Spanish translation, but even that is just a very small bit of Pext. At least for now, I want more of an "I'm the admin, others are mods" structure. Like OPs and half-ops on IRC, you know?

Reply:
Sure, and that's fine. I have made the necessary corrections, or at least I hope that the changes I've made reflect what you wanted to see.